### PR TITLE
update readme example

### PR DESCRIPTION
**Pull request by VladStepanov**
Promises are chainable, so it shouldn't be nested like now in example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,12 +77,14 @@ const img = document.getElementById('img')
 
 // Load model from my S3.
 // See the section hosting the model files on your site.
-nsfwjs.load().then(function (model) {
-  model.classify(img).then(function (predictions) {
+nsfwjs.load()
+  .then(function (model) {
+    return model.classify(img)
```
Comment (GantMan) [inline README.md:82]: looks good 👍

File: README.md [hunk 2]
```diff
@@ -77,12 +77,14 @@ const img = document.getElementById('img')
...
+  })
+  .then(function (predictions) {
     // Classify the image
```
Comment (GantMan) [inline README.md:85]: this comment seems out of place, too.  Not your fault, but could you move it?  Technically, by the time it gets here the image has been classified.

**Review by GantMan — CHANGES_REQUESTED** [review]
Looks great.  Can you make the one small change I recommend?

File: README.md (hunk 2 above)
Comment (VladStepanov) [reply README.md]: Moved
Really overlooked the comment

**Review by GantMan — APPROVED** [review]

**Comment by GantMan** [comment]
Thanks for contributing!

**Comment by GantMan** [comment]
You're added to the wall on the readme

**Pull request merged**